edge-hub/watchdog/src/main.rs

```diff
@@ -0,0 +1,85 @@
+mod child;
+
+use std::{
+    io::Error,
+    sync::atomic::{AtomicBool, Ordering},
+    sync::Arc,
+    thread,
+};
+
+use anyhow::{Context, Result};
+use child::run;
+use signal_hook::{iterator::Signals, SIGINT, SIGTERM};
+use tracing::{error, info, subscriber, Level};
+use tracing_subscriber::fmt::Subscriber;
+
+fn main() -> Result<()> {
+    init_logging();
+    info!("Starting watchdog");
+
+    let should_shutdown = register_shutdown_listener()
+        .context("Failed to register sigterm listener. Shutting down.")?;
+
+    let edgehub_handle = run(
+        "Edge Hub".to_string(),
```

Comment:
somewhere in logs, you put "edgehub", somewhere else "Edge Hub". lets add some consistency across watchdog logs? same comment for broker

Reply:
The way I meant to do this is "Edge Hub" and "MQTT Broker". I'll fix the offending instances